### PR TITLE
Update ovn certs for openstack-hyervisor snap

### PR DESCRIPTION
**Pull request by hemanthnakkina**
Run vault action to generate certificates and
update corresponding openstack-hypervisor configs.

### CODE REVIEW

File: sunbeam/commands/ohv.py [hunk 1]
```diff
@@ -156,11 +202,45 @@ def __init__(self):
...
+                "action_cmd": "get-southbound-db-url",
+                "action_params": {},
+                "map_alias": {"url": "ovn-sb-connection"},
+                "attributes_to_update": ["ovn-sb-connection"],
+            },
+            {
+                "app": "vault",
+                "action_cmd": "generate-certificate",
+                "action_params": {
+                    "cn": "nova-hypervisor",
```
Comment (javacruft) [inline sunbeam/commands/ohv.py:220]: cn probably wants to be unique - I've used the hostname in testing.

As sans are purely IP address based this is important to have in the configuration

Comment (javacruft) [reply sunbeam/commands/ohv.py:220]: The alternative is to add the FQDN into the list of sans - the server side code should deal with this.

File: sunbeam/utils.py [hunk 2]
```diff
@@ -67,3 +71,35 @@ def parse_version(version: str) -> VersionInfo:
...
+        ]
+        addresses.extend(address)
+
+    if "127.0.0.1" in addresses:
+        addresses.remove("127.0.0.1")
+
+    return addresses
+
+
+def encode_tls(cert_or_key: str) -> bytes:
```
Comment (javacruft) [inline sunbeam/utils.py:93]: I think the return type is str not bytes

**Comment by javacruft** [comment]
As a more general comment - 'nova-hypervisor' is not unique and although that's OK for now I'd suggest we move to per-hostname naming for usernames - this isolates the account to each host and makes lifecycle management easier in the future - we can actually do things like cleanup accounts when nodes are removed from the deployment

File: sunbeam/commands/ohv.py (hunk 1 above)
Comment (hemanthnakkina) [reply sunbeam/commands/ohv.py]: Fixed, sans is made as part of snap config and updated as part of init (for now).

**Review by javacruft — APPROVED** [review]

**Pull request merged**